src/app/components/home-page/home-page.component.ts

```diff
@@ -0,0 +1,25 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-home-page',
+  templateUrl: './home-page.component.html',
+  styleUrls: ['./home-page.component.scss']
+})
+export class HomePageComponent implements OnInit {
+
+  public dashboardURL = 'http://ec2-3-228-127-99.compute-1.amazonaws.com/d-solo/ZEcnMHoWk/lantern?orgId=1&refresh=10s&theme=light';
+
+  public totalEndpoints = this.dashboardURL + '&panelId=14';
+  public success = this.dashboardURL + '&panelId=12';
+  public unsuccessful = this.dashboardURL + '&panelId=16';
+  public unresponsive = this.dashboardURL + '&panelId=18';
+  public newEndpoints = this.dashboardURL + '&panelId=20';
+  public responseTimeRange = this.dashboardURL + '&panelId=22';
+  public currentResponseTime = this.dashboardURL + '&panelId=8';
```

Comment:
I think so, the only tests I can think of that make sense to me are making sure they get loaded into the html properly and that they reference valid panel ids. Was there anything else you had in mind?

Reply:
I wasn't able to write a test to see if they have valid panel ids because you apparently can't capture iframe errors since it's a cross-domain request? It seems like a good idea to check if these urls return errors but i didn't see a great way to do that, at least at first glance.